src/Tribe/Tickets_Handler.php

```diff
@@ -124,6 +124,58 @@ public function print_checkedin_totals() {
 	}
 
 	/**
+	 * Get the total capacity event.
+	 *
+	 * @param $event_id int (null)
+	 *
+	 * @since TBD
+	 *
+	 * @return int number of tickets ( -1 means unlimited )
+	 */
+	public function get_total_event_capacity( $post = null ) {
+		$post_id = Tribe__Main::post_id_helper( $post );
+
+		if ( ! $post_id ) {
```

Comment:
Oh! You are right, [it does](https://github.com/moderntribe/tribe-common/blob/c092a7b79ab341a737936cd049c7d40a6701bedb/src/Tribe/Main.php#L441). That's handy!